grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientFactory.java

```diff
@@ -91,6 +94,13 @@ public Object newClient(ClientBuilderParams params) {
 
         final HttpClient httpClient = newHttpClient(params);
         final Method stubFactoryMethod = findStubFactoryMethod(clientType, stubClass);
```

Comment:
Not necessarily related to this PR but how about renaming `stubClass` to `enclosingClass`? Stub class is `clientType` actually.

Reply:
Yeah, sounds good!